user-ms/src/main/kotlin/puc/config/SecurityConfig.kt

```diff
@@ -20,6 +20,7 @@ class SecurityConfig(
 
     private val permitAllEndpoints = arrayOf(
         "/users/register",
+        "/users/register/admin",
```

Comment:
não faz sentido uma rota /admin

se for para implementar a criação de usuário admin é necessário utilizar a rota já existente /register e adicionar autenticação opcional nela, e quando tentar criar com a role user no body, irá verificar se o usuário logado possui a role admin